torch/_prims/context.py

```diff
@@ -181,7 +181,11 @@ def _is_func_unsupported_nvfuser(torch_function_mode, func, args, kwargs):
     with torch.overrides.enable_torch_function_mode(
         torch_function_mode, replace=torch_function_mode.inner
     ):
-        gm = get_isolated_graphmodule(func, args, kwargs)
+        g = torch._C._AutoDispatchBelowAutograd()
```

Comment:
nitpick: Is `get_isolated_graphmodule` supposed to fail? if not, a context manager would probably looks cleaner for _AutoDispatchBelowAutograd?

Reply:
Something was breaking with incorrectly setting the `requires_grad` attribute with `make_fx` in `test_batch_norm_backward_nvprims`, but now it seems it got fixed. So I'll remove this change.